src/main/java/com/google/sps/utility/JsonUtility.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.utility;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.http.HttpServletResponse;
+
+/** Utility class for JSON operations. */
+public final class JsonUtility {
+
+  /** Prevent instances of this class. */
+  private JsonUtility() {}
+
+  /**
+   * Sends data to a HTTP Servlet Response.
+   *
+   * @param response HTTP Servlet Response that data is sent to.
+   * @param object Contains the data that is to be sent.
+   * @throws IOException
+   */
+  public static void sendJson(HttpServletResponse response, Object object) throws IOException {
+    Gson gson = new Gson();
+    String json = gson.toJson(object);
+    response.getWriter().println(json);
```

Comment:
Just curious - does this work? 

You are writing to the response object (which is magically handled within the servlet), but I don't think that "magic" would work here since you aren't returning the response object (you're essentially just writing to a response object and then disposing of it at the end of the function).

I think it would suffice to return the JSON string

Reply:
As an add-on, this method does not enforce that the response is set to be of type JSON. This could lead to errors if misused

Reply:
At least in all the cases where I have used it in my starter projects, writing to a response object and disposing it did exactly what I wanted it to do but feel free to let me know if there's a reason why returning a JSON string would be ideal. Also, you having to question whether this works is on me for not writing tests before requesting review - sorry! I'll add one soon. I also agree that the content type should be set.